var/spack/repos/builtin/packages/rocksdb/package.py

```diff
@@ -22,6 +23,7 @@ class Rocksdb(MakefilePackage):
 
     variant('bz2', default=False, description='Enable bz2 compression support')
     variant('lz4', default=True, description='Enable lz4 compression support')
+    variant('shared', default=True, description='Build shared library')
```

Comment:
You may want to add a conflict for `~shared~static` since that combination is presumably not possible.

Reply:
Good point. :smile: Fixed.